src/Components/Shared/Table/table.module.css

```diff
@@ -0,0 +1,73 @@
+.container {
+  margin-bottom: 50px;
+}
+
+.noneTrainer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 40vw;
+  height: 50px;
+  border-radius: 5px;
+  background-color: #d9d9d9ea;
+  font-size: 30px;
+}
+
+table {
+  border-collapse: separate;
+  border-spacing: 0;
+  width: 1044px;
+  border: 1px solid rgba(128, 128, 128, 0.63);
+  border-radius: 5px;
+}
+
+tr {
+  height: 48px;
+}
+td {
+  font-size: 16px;
+  text-align: center;
+}
+
+thead tr th:first-child,
+tbody tr td:first-child {
+  text-align: start;
+}
```

Comment:
i think you should declare at least a className before to call a tag, if we use some thead or others tag in other place and we could have a problems with styles, to avoid that it would be better use at least a  container tag with a className